grammars/erlang.bnf

```diff
@@ -321,6 +324,32 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= empty '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
```

Comment:
Could you suggest any alternatives?

Now the situation is: we have a bug in parser generator and we cannot fix it now. 
If we pin on `{`, this bug will add redundant error text.
We will definitely pin on `{` because of tuple_expression element type. I don't want to remove this piece of code (`{elementType=tuple_expression}`) because formatting will be messed up.
So, in my opinion, the simplest solution is to add another element to pin on it. Empty is great.
In addition, anyone else could not add another rule because app file contains of only one term. Yes, maybe some day language rules will be changed. But some day I hope the bug will be fixed.